api/DOMMatrixReadOnly.json

```diff
@@ -2068,6 +2068,54 @@
           }
         }
       },
+      "translate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMMatrixReadOnly/translate",
+          "support": {
+            "chrome": {
+              "version_added": "61"
```

Comment:
This [landed in 45](https://storage.googleapis.com/chromium-find-releases-static/2b3.html#2b3ee51b8aa11c3fcb57ca34abdf65929d0f4e0b).

Reply:
Thank you - I've updated these now